### ISSUE TITLE
Component: Title

### ISSUE THREAD

**Issue by zhiying99** [primefaces-extensions/primefaces-extensions#239]
**Describe the bug**
p:menubar (p:menu, p:meneButton as well) is not accessable. That is, when tabbing through the menu using a screen reader like JAWS, it does not read any of the menu items, even all the generated <a/> elements have titles or aria-lables. 

**Reproducer**
See here: https://www.primefaces.org/showcase/ui/menu/menubar.xhtml

**To Reproduce**
Open a screen reader (JAWS for example), tab through "File" "Edit" menus and submenus, the screen reader will read nothing.

**Expected behavior**
It would be nice to have an unified focus and screen reader interaction behaviour, like p:tabMenu, which works perfectly.   

**Sample XHTML**
It seems that the focus is always landed on the surrounding elements like <div tabindex="0" class="ui-helper-hidden-accessible"></div>, or <li … />, while the accessable values are inside the inner <a/> elements. However, simply adding a title or aria-label to <li … /> does not work either.

**Comment by jepsar** [comment]
@zhiying99 why did you open an issue at PrimeFaces Extensions and not at https://github.com/primefaces/primefaces/issues ?

**Comment by zhiying99** [comment]
sorry, an error, can I just "Close issue"?

**Comment by jepsar** [comment]
@zhiying99 yes, and please set a title on the new issue

**Comment by zhiying99** [comment]
ok, thank you!
